Src/LinearSolvers/MLMG/AMReX_MLABecLaplacian.cpp

```diff
@@ -101,6 +108,16 @@ MLABecLaplacian::setScalars (Real a, Real b) noexcept
     }
 }
 
+/**
+ * Sets alpha as a scalar field to values from a single component
+ * mutlifab.
+ *
+ * \param [in] amrlev The level of the multifab for the solver, with
+ *                    \p amrlev = 0 always being the lowest level in the
+ *                    AMR hierarchy represented in the solve.
+ * \param [in] alpha  Constant value to populate across mutlifab.
```

Comment:
What does *Constant* mean here?

Reply:
Looks like this was meant for the other version of setACoeffs, not this one.

Reply:
I was thinking "scalar constant", as in the statement of the equation. However, in the context of coding I can see how calling it a constant would be problematic. Do you think changing all the "constant" to "scalar constant" is enough to avoid that ambiguity?

Reply:
I think this is a copy-paste error.  This is what's in the PR
```
\param [in] beta Multifab for beta values.
MLABecLaplacian::setBCoeffs (int amrlev, const Array<MultiFab const*,AMREX_SPACEDIM>& beta)

\param [in] beta   Constant value to populate across mutlifab.
MLABecLaplacian::setBCoeffs (int amrlev, Real beta)

\param [in] alpha  Constant value to populate across mutlifab.
MLABecLaplacian::setACoeffs (int amrlev, Real alpha)

\param [in] alpha  Constant value to populate across mutlifab.
MLABecLaplacian::setACoeffs (int amrlev, const MultiFab& alpha)
```

Reply:
Yes, thanks for catching that!

Reply:
Nevermind, I see it now. I'll fix the description.